dotnet/src/InternalUtilities/src/Diagnostics/Verify.cs

```diff
@@ -11,31 +11,50 @@
 namespace Microsoft.SemanticKernel;
 
 [ExcludeFromCodeCoverage]
-internal static class Verify
+internal static partial class Verify
 {
-    private static readonly Regex s_asciiLettersDigitsUnderscoresRegex = new("^[0-9A-Za-z_]*$");
-    private static readonly Regex s_filenameRegex = new("^[^.]+\\.[^.]+$");
+#if NET7_0_OR_GREATER
```

Comment:
Some other places for `[GenerateRegex]` uses `#if NET` without versioning but this one is specific to 7. What is the only available above 7 in here?

Reply:
If memory serves, this shared file is built into one or more projects that have a net6.0 tfm.

Reply:
It looks like I removed that usage, so I changed this to just `#if NET`.